Sources/AccessibilitySnapshot/Core/Swift/Classes/UIAccessibility+SnapshotAdditions.swift

```diff
@@ -22,7 +22,7 @@ extension NSObject {
     func accessibilityDescription(context: AccessibilityHierarchyParser.Context?) -> (description: String, hint: String?) {
         var accessibilityDescription = accessibilityLabelOverride(for: context) ?? accessibilityLabel ?? ""
 
-        var hintDescription = hidesAccessibilityHint() ? nil : accessibilityHint?.nonEmpty()
```

Comment:
I think we still need this since there are reasons that could be true other than the OS version.

Reply:
I can restore it, but if I'm understanding the logic correctly, the implementation of that check would now just return `false`, right?
This:
```swift
    private func hidesAccessibilityHint() -> Bool {
        if NSObject.majorOSVersion >= 13 {
            return false
        } else {
            return accessibilityTraits.contains(.tabBarItem)
        }
    }
```
becomes:
```swift
    private func hidesAccessibilityHint() -> Bool {
        false
    }
```
if we're always building for iOS 13 and above.

Reply:
Oh sorry, GitHub was showing the diff weird and it looked like there was more in that method. You're right.